examples/attribute-transition/README.md

```diff
@@ -0,0 +1,7 @@
+<div align="center">
+   <img width="150" heigth="150" src="https://webpack.js.org/assets/icon-square-big.svg" />
+</div>
+
+## Attribute Transition
+
+Demonstrate the abilities to interpolate attribute arrays.
```

Comment:
To reduce confusion for external users who tend to go directly to master, we are creating new examples in the `examples/wip` folder.

Something to think about: 
* While we want simple examples, when we have integrated a new feature so cleanly into the API it would be natural to start using it / showing it off in other (existing or new) examples to make them more compelling. Since the API is so simple it shouldn't make those examples less accessible. It would also show that this feature handles more corner cases, I am still uncertain about exactly where the limitations land in practice. 
* Also, we have more animation features coming in, do they all get separate examples, or could we keep building on this one?